include/SubPanel/SubPanelTiles.php

```diff
@@ -443,7 +443,6 @@ public function get_buttons($thisPanel, $panel_query=null)
         $widget_contents = smarty_function_sugar_action_menu(
             [
                 'buttons' => $buttons,
-                'flat' => !empty($thisPanel->_instance_properties['flat']),
```

Comment:
This was meant for displaying action menu with flat buttons instead of dropdown options (see screenshot)... it was a useful feature, why was this removed?

This was possible adding `'flat' => true,` in subpanel's definition.

![imagen](https://user-images.githubusercontent.com/5542756/69733501-2bc79000-110c-11ea-8a8d-7e5df4300d4b.png)

Reply:
@Dillon-Brown @cameronblaikie when you have the time can you please check this out, we were using this feature in production. Thanks!